ffv1.md

```diff
@@ -1105,37 +1120,35 @@ initial\_state[ i ][ j ][ k ] = ( pred + initial\_state\_delta[ i ][ j ][ k ] )
 
 ## Quantization Tables
 
-The quantization tables are stored by storing the number of equal entries -1 of the first half of the table using the method described in [Range Non Binary Values](#range-non-binary-values). The second half doesn’t need to be stored as it is identical to the first with flipped sign.
+The quantization tables are stored by storing the number of equal entries minus one of the first half of the table using the method described in [Range Non Binary Values](#range-non-binary-values). The second half doesn’t need to be stored as it is computed following the formula below.
 
 example:
 
-Table: 0 0 1 1 1 1 2 2-2-2-2-1-1-1-1 0
+Table: 0 0 1 1 1 1 2 2 -2 -2 -2 -1 -1 -1 -1 0
 
 Stored values: 1, 3, 1
 
 ```c
 function                                                      | type
 --------------------------------------------------------------|-----
-QuantizationTable( i ) {                                      |
+QuantizationTablePerSet( i ) {                                |
     scale = 1                                                 |
-    for( j = 0; j < MAX_CONTEXT_INPUTS; j++ ) {               |
+    for( j = 0; j < 5; j++ ) {                                |
         QuantizationTablePerContext( i, j, scale )            |
         scale *= 2 * len_count[ i ][ j ] - 1                  |
     }                                                         |
     context_count[ i ] = ( scale + 1 ) / 2                    |
 }                                                             |
 ```
 
-MAX\_CONTEXT\_INPUTS is 5.
-
 ```c
 function                                                      | type
 --------------------------------------------------------------|-----
 QuantizationTablePerContext(i, j, scale) {                    |
     v = 0                                                     |
     for( k = 0; k < 128; ) {                                  |
-        len - 1                                               | sr
-        for( a = 0; a < len; a++ ) {                          |
+        len_minus1                                            | ur
```

Comment:
it is not a counter, it is a temporary value, so not defined. It is a bit explained in the intro ("storing the number of equal entries -1"), does it need a dedicated explicit definition? as it is temporary, I suggest not to do it.